backend/benefit/applications/api/v1/views.py

```diff
@@ -154,7 +154,23 @@ def simplified_application_list(self, request):
         )
 
         serializer = self.serializer_class(qs, many=True, context=context)
-        return Response(serializer.data, status=status.HTTP_200_OK)
+
+        order_by = request.GET.get("order_by")
+        is_order_descending = True if order_by and order_by.lower() == "desc" else False
+
+        order_key = (
+            request.GET.get("order_key")
+            if request.GET.get("order_key")
+            else "created_at"
```

Comment:
Here, you can use `.get()` method's default parameter, for example `request.GET.get("order_key", "created_at")`.